src/components/NetworkSettings/NetworkStatusBar.js

```diff
@@ -5,26 +5,46 @@
  * LICENSE file in the root directory of this source tree.
  */
 import { useSelector } from 'react-redux';
-import { eq } from 'lodash';
+import { isEqual } from 'lodash';
 import { t } from 'ttag';
 import { AlertUI } from '../../styles/themes';
 import { ToplineBar } from '../ToplineBar';
 import { PRE_SETTINGS_MAINNET } from '../../constants';
 
 const customNetworkText = t`Custom network`;
 
-export const NetworkStatusBar = () => {
-  const networkSettings = useSelector((state) => state.networkSettings);
-  if (eq(networkSettings, PRE_SETTINGS_MAINNET)) {
-    return null;
+function notMainnet(networkSettings) {
```

Comment:
I think this method deserves a comment explaining why you split the validation in two.

Reply:
Added: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/360/commits/f31c263e2c8a64ee37f553dcd1dca0908b155267